app/components/option_box/index.tsx

```diff
@@ -32,34 +35,55 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
         justifyContent: 'center',
         minWidth: 80,
     },
+    destructiveContainer: {
+        backgroundColor: changeOpacity(theme.dndIndicator, 0.04),
+    },
     text: {
         color: changeOpacity(theme.centerChannelColor, 0.56),
         paddingHorizontal: 5,
         ...typography('Body', 50, 'SemiBold'),
     },
 }));
 
-const OptionBox = ({activeIconName, activeText, containerStyle, iconName, isActive, onPress, testID, text}: OptionBoxProps) => {
+const OptionBox = ({
+    activeIconName,
+    activeText,
+    containerStyle,
+    iconName,
+    isActive,
+    onPress,
+    testID,
+    text,
+    destructiveIconName,
+    destructiveText,
+    isDestructive,
+}: OptionBoxProps) => {
     const theme = useTheme();
     const [activated, setActivated] = useState(isActive);
     const styles = getStyleSheet(theme);
+
     const pressedStyle = useCallback(({pressed}: PressableStateCallbackType) => {
-        const style = [styles.container, Boolean(containerStyle) && containerStyle];
+        const style = [
+            styles.container,
+            Boolean(containerStyle) && containerStyle,
```

Comment:
Nit: react should handle well being passed undefined as a style, so you can directly pass `containerStyle`.